### PR TITLE
completed javascript-loops exercise - Colin Macrae

### PR DESCRIPTION
**Pull request by colin-macrae**

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+// console.log(logEachCharacter('hello'));
+
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log(string[i]);
+  }
+}
+
+// function doubleAll(numbers) {
```
Comment (shawnkost) [inline javascript-loops/main.js:48]: This function shouldn't be commented out!

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+//   }
+//   return doubled;
+// }
+
+// const abc = { a: 1, b: 2, c: 3 };
+
+function getKeys(object) {
+  var keys = [];
+  for (const property in object) {
+    keys.push([property]);
```
Comment (shawnkost) [inline javascript-loops/main.js:61]: This should just be `keys.push(property)` without the brackets

**Review by shawnkost — CHANGES_REQUESTED** [review]
A couple changes Colin!

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**